recipe/recipe.yaml

```
@@ -37,71 +34,93 @@ source:
         - fix-argminmax.patch
 
 build:
-  number: 0
-  skip: not (match(python, python_min ~ ".*") and is_abi3)
-  python:
-    version_independent: true
+  number: ${{ build_number }}
 
-requirements:
-  build:
-    - if: build_platform != target_platform
-      then:
+outputs:
+  - package:
+      name: polars
+    build:
+      script: echo "No build step is needed"
+      string: ${{ polars_variant | replace('-', '_') }}_h${{ hash }}_${{ build_number }}
+      variant:
+        down_prioritize_variant: ${{ 0 if polars_variant == 'default' else 1 }}
+    requirements:
+      run:
+        - ${{ pin_subpackage('polars-' ~ polars_variant, exact=True) }}
```

```suggestion
        - ${{ pin_subpackage('polars-' ~ polars_variant, exact=true) }}
```
i find `true` prettier than `True`; yaml is not python